backend/controllers/orderController.js

```
@@ -17,23 +26,96 @@
     if (!mongoose.isValidObjectId(userId)) {
       return res
         .status(400)
         .json({ success: false, message: "Invalid userId" });
     }
+    if (!Array.isArray(items) || items.length === 0) {
+      return res
+        .status(400)
+        .json({ success: false, message: "Items must be a non-empty array" });
+    }
+
+    for (let item of items) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'item' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)